programs/mpl-core/src/plugins/lifecycle.rs

```diff
@@ -795,6 +795,8 @@ pub(crate) struct PluginValidationContext<'a, 'b> {
     pub resolved_authorities: Option<&'b [Authority]>,
     /// The new owner account.
     pub new_owner: Option<&'a AccountInfo<'a>>,
+    /// The new authority address.
+    pub new_authority: Option<&'b Pubkey>,
```

Comment:
with all these optional fields, it might be time (in a different PR) to split validation contexts into different ones so it's predictable

Reply:
Makes sense! I've always disliked the unclear Target and optional new_x naming.